ArjunaJTA/jta/tests/classes/com/hp/mwtests/ts/jta/commitmarkable/TestCommitMarkableResourceFailActivate.java

```diff
@@ -154,7 +154,6 @@ public void run() {
 
 		// Run the scan to clear the content
 		manager.scan();
-		manager.scan();
```

Comment:
Prior to this change the list of XAResources would need refreshing from getNewXAResource function. This is no longer required to the CMR records will be garbage collected a cycle earlier